Firestore/core/src/firebase/firestore/util/executor.h

```diff
@@ -35,11 +35,16 @@ class DelayedOperation {
   DelayedOperation() {
   }
 
+  explicit operator bool() const {
```

Comment:
This needs a comment about what it means. Secondarily I'd consider giving this method a name so that it's clear in the caller what's going on. As it stands I have no idea what this represents.

Reply:
Added a comment. I'm mirroring `std::function` interface (though to be fair, a similar check for `std::future` is a named member function, `valid`). Also, it's more similar to other platforms, where pretty much any object can be compared to `null`/`nil`/similar, rather than having a named method for the check. If you'd prefer a named method, how about `IsValid`? `IsEmpty`?